text/0050-fellowship-salaries.md

```diff
@@ -0,0 +1,118 @@
+# RFC-0050: Fellowship Salaries
+
+|                 |                                                                               |
+| --------------- | ----------------------------------------------------------------------------- |
+| **Start Date**  | 15 November 2023                                                              |
+| **Description** | Proposal to set rank-based Fellowship salary levels.                          |
+| **Authors**     | Joe Petrowski, Gavin Wood                                                     |
+
+## Summary
+
+The Fellowship Manifesto states that members should receive a monthly allowance on par with gross
+income in OECD countries. This RFC proposes concrete amounts.
+
+## Motivation
+
+One motivation for the Technical Fellowship is to provide an incentive mechanism that can induct and
+retain technical talent for the continued progress of the network.
+
+In order for members to uphold their commitment to the network, they should receive support to
+ensure that their needs are met such that they have the time to dedicate to their work on Polkadot.
+Given the high expectations of Fellows, it is reasonable to consider contributions and requirements
+on par with a full-time job. Providing a livable wage to those making such contributions makes it
+pragmatic to work full-time on Polkadot.
+
+Note: Goals of the Fellowship, expectations for each Dan, and conditions for promotion and demotion
+are all explained in the Manifesto. This RFC is only to propose concrete values for allowances.
+
+## Stakeholders
+
+- Fellowship members
+- Polkadot Treasury
+
+## Explanation
+
+Although the Manifesto (Section 8) specifies a monthly allowance in DOT, this RFC proposes the use
+of USDT instead. The allowance is meant to provide members stability in meeting their day-to-day
+needs and recognize contributions. Using USDT provides more stability and less speculation.
+
+This RFC proposes the following yearly salary levels in USDT, based on rank:
+
+| Dan  | Annual Salary |
+|:----:|:-------------:|
+|    I |     10,000    |
+|   II |     20,000    |
+|  III |     80,000    |
+|   IV |    120,000    |
+|    V |    160,000    |
+|   VI |    200,000    |
+|  VII |    200,000    |
+| VIII |    200,000    |
+|   IX |    200,000    |
+
+Note that there is a sizable increase between II Dan (Proficient) and III Dan (Fellow). By the third
+Dan, it is generally expected that one is working on Polkadot as their primary focus in a full-time
+capacity. The salary at this level is commensurate with average salaries in OECD countries (note:
+77,000 USD in the U.S., with an average engineer at 100,000 USD).
```

Comment:
Can we make it a bit more technical, to allow for easy evaluation and adjustments?  

It looks like the 100K are in the center between DAN 3 and 4. Then we can use that as baseline and calculate the other ones in relation to that. Eg DAN 5 is DAN 4 +33% and DAN 2 is -75% of DAN 3 and so on.

Reply:
I'm hesitant to make it more technical because I think statistics in this case are a facade for the actual numbers. There are just so many data sources on salaries that you could always cherry pick data or aggregation methods that support the number you want. Then it appears technical because you have lots of math/data and "precise" numbers, but it's really not.

For example, I was not able to find OECD sourced data on _engineering_ salaries, just on general averages. Even for engineering specific data across large geopolitical areas (America, Europe), it varies widely based on field and location. I found a few data sources that indicated US engineering salaries were around 100k on average and in the 80-160k range generally.

I would be in favor of altering the structure such that the levels are all relative to each other. E.g., everything is relative to III Dan (so I Dan is 12.5%, II Dan is 25%, etc.), and then make a proposal for a single number: the III Dan level, which people agree represents a reasonable salary for an IC.

Reply:
Makes everything relative to the III Dan level will be good. It simplifies future work when we need to adjust the amount. It is easier to propose say change the base to a new number, than try to figure out the bunch numbers for every rank.